### PR TITLE
Update nginx security

### PR DESCRIPTION
**Pull request by Roust**
Update nginx cipher suites for more security
Update HSTS configuration syntax

### CODE REVIEW

File: yunohost-config-nginx/config/template.conf [hunk 1]
```diff
@@ -26,8 +26,8 @@ server {
     ssl_session_cache shared:SSL:50m;
     ssl_prefer_server_ciphers on;
     ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-    ssl_ciphers ALL:!aNULL:!eNULL:!LOW:!EXP:!RC4:!3DES:+HIGH:+MEDIUM;
-    add_header Strict-Transport-Security "max-age=31536000;";
+    ssl_ciphers 'AES256+EECDH:AES256+EDH:AES128+EECDH:AES128+EDH:+SHA';
```
Comment (taziden) [inline yunohost-config-nginx/config/template.conf:29]: I'd suggest using these ciphers  :
`ssl_ciphers HIGH:!aNULL:!MD5:!3DES;`
They also give an A+ on ssllabs and doesn't require much work to ensure secure ciphers in the future (using HIGH parameter). Example :  https://www.ssllabs.com/ssltest/analyze.html?d=ilico.org

**Comment by Roust** [comment]
sry je passe en FR, mon niveau d'anglais est trop mauvais pour m'explimer.

Avec HIGH, on ne force pas l'utilisation de Diffie Hellman (DHE, ECDHE) et on utilise CAMELIA que je ne connais pas, donc auquel je n'ai pas complètement confiance. D'où cette proposition assez stricte, j'avoue.

**Comment by taziden** [comment]
Your proposal seems good to me, anyway. 
Let's increase YNH security ;)

**Comment by jeromelebleu** [comment]
Hi!

This repo is not used anymore. Please use this one for your pull requests regarding nginx: https://github.com/YunoHost/yunohost-config-nginx

And I agree that we should set a proper ssl_ciphers such as those suggested by https://mozilla.github.io/server-side-tls/ssl-config-generator/

**Pull request closed without merging**